sdk-api-src/content/bcrypt/nf-bcrypt-bcryptimportkeypair.md

```diff
@@ -51,7 +51,7 @@ api_name:
 
 ## -description
 
-The <b>BCryptImportKeyPair</b> function imports a <a href="/windows/desktop/SecGloss/p-gly">public/private key pair</a> from a key <a href="/windows/desktop/SecGloss/b-gly">BLOB</a>. The <a href="/windows/desktop/api/bcrypt/nf-bcrypt-bcryptimportkey">BCryptImportKey</a> function is used to import a <a href="/windows/desktop/SecGloss/s-gly">symmetric key</a> pair.
+The <b>BCryptImportKeyPair</b> function imports a <a href="/windows/desktop/SecGloss/p-gly">public/private key pair</a> from a key <a href="/windows/desktop/SecGloss/b-gly">BLOB</a>. The <a href="/windows/desktop/api/bcrypt/nf-bcrypt-bcryptimportkey">BCryptImportKey</a> function is used to import a <a href="/windows/desktop/SecGloss/s-gly">symmetric key</a>.
```

Comment:
This change looks good, but I try to update "desktop" to "win32" whenever PRs come through to avoid redirects to the updated path.
```suggestion
The <b>BCryptImportKeyPair</b> function imports a <a href="/windows/win32/SecGloss/p-gly">public/private key pair</a> from a key <a href="/windows/win32/SecGloss/b-gly">BLOB</a>. The <a href="/windows/win32/api/bcrypt/nf-bcrypt-bcryptimportkey">BCryptImportKey</a> function is used to import a <a href="/win32/desktop/SecGloss/s-gly">symmetric key</a>.
```